cockpit/package.json

```diff
@@ -41,5 +43,8 @@
     "style-loader": "^0.18.2",
     "webpack": "^2.6.1"
   },
-  "dependencies": {}
+  "dependencies": {
+    "bootstrap": "^4.1.1",
+    "bootstrap-less": "^3.3.8"
+  }
```

Comment:
Do we really need to add these libraries to `dependencies` and not only to `devDependencies`?

Reply:
I copied what cockpit-podman does https://github.com/cockpit-project/cockpit-podman/blob/master/package.json
@martinpitt @Gundersanne do you know why it's under `dependencies` rather than `devDependencies` in the cockpit-podman source?

Reply:
I've put this under devDependencies now.